packages/serviceplaceholder-adapter/test/adapter.test.ts

```diff
@@ -28,6 +28,7 @@ import { DEFAULT_CONFIG } from '../src/config'
 import { ADAPTER_NAME } from '../src/constants'
 // TODO update mock file -
 // for fetch: run fetch with trace-level logs:
+// [Please make sure to first set the strategy to 'full' in the DEFAULT_LOGGING_CONFIG under logging.utils.ts]
```

Comment:
Thanks!
Can you please just put the needed configuration here ?

Reply:
no, I meant copy-paste it here in the comment so it would be easy to copy-paste from the comment to my local config file

Reply:
something like (not sure I wrote here the actual accurate config) - 
```suggestion
// add this to you adapter.nacl file to capture full logs of the responses:
  client = {
    logging = {
      responseStrategies = [
        {
          strategy = "full"
        },
      ]
    }
  }
 //
```